ValetTouchIDTest/ValetTouchIDTestViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ValetTouchIDTestViewController.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/20/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+
+import UIKit
+
+
+final class ValetTouchIDTestViewController : UIViewController
+{
+    // MARK: Properties
+
+    @IBOutlet var textView : UITextView?
+    let secureEnclaveValet : VALSecureEnclaveValet = VALSecureEnclaveValet.init(identifier: "UserPresence", accessControl: VALAccessControl.UserPresence) as VALSecureEnclaveValet!
+    let username : String = "CustomerPresentProof"
+
+
+    // MARK: Outlets
+
+    @IBAction func setOrUpdateItem(sender: UIResponder)
+    {
+        let stringToSet = "I am here! " + NSUUID.init().UUIDString
+        let setOrUpdatedItem = secureEnclaveValet.setString(stringToSet, forKey: username)
+        updateTextView([#function, (setOrUpdatedItem ? "Success" : "Failure")])
+    }
+
+    @IBAction func getItem(sender: UIResponder)
+    {
+        var userCancelled: ObjCBool = false
```

Comment:
Won't compile as an inout Swift Bool. This was the number one pain point I saw for sure